src/components/Pages/Contact.vue

```diff
@@ -24,7 +24,7 @@
                   hint="For example: email address, Telegram handle, phone number"
                 />
                 <v-select
-                  :items=items
+                  :items="items"
```

Comment:
For future reference, please keep changes in a PR focused on a single topic. Unrelated code quality or formatting fixes should be submitted separately, so changes are easier to review and track. As this is a small change and doesn't hinder the review, I'll let it slide!